hooks/post-command

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+if [ "$BUILDKITE_COMMAND_EXIT_STATUS" = "0" ]; then
+  echo "command exited 0; exiting"
+  exit 0
+fi
+
+if [ -z "$BUILDKITE_TOKEN" ]; then
+  echo "BUILDKITE_TOKEN not provided; exiting"
+  exit 0
+fi
+
+if_condition="$BUILDKITE_PLUGIN_BAIL_EARLY_IF"
+
+if [ -n "$if_condition" ]; then
+  if [[ "$if_condition" =~ \] ]]; then
+    echo "'if' condition cannot contain brackets; exiting"
+    exit 0
+  fi
+
+  if eval "[ $if_condition ]"; then
+    echo "'if' condition evaluated true; continuing"
```

Comment:
could this get noisy?

Reply:
it shows up in a collapsed section of the logs, so only if you open it up and are looking in there :P

<img width="916" alt="Screen Shot 2020-07-14 at 1 04 26 PM" src="https://user-images.githubusercontent.com/4969439/87471042-8eee3000-c5d2-11ea-88de-647de7f57855.png">